inc/template-tags.php

```diff
@@ -448,12 +448,12 @@ function twentytwenty_body_classes( $classes ) {
 		}
 
 		// Check whether the current page should have an overlay header.
-		if ( is_page_template( array( 'template-cover.php' ) ) ) {
+		if ( is_page_template( array( 'templates/template-cover.php' ) ) ) {
```

Comment:
Did you try this? Does that function return a path or just a name?

Reply:
It simply returns true, if the page was found, or false, if the page wasn't found: https://developer.wordpress.org/reference/functions/is_page_template/#return

Reply:
According to that code, it would never match because it's looking at the slug, not the path.